mono/mini/interp/transform.c

```diff
@@ -4160,6 +4165,13 @@ mono_interp_transform_method (RuntimeMethod *runtime_method, ThreadContext *cont
 		offset += size;
 	}
 	offset = (offset + 7) & ~7;
+
+	runtime_method->exvar_offsets = g_malloc (header->num_clauses * sizeof (guint32));
+	for (i = 0; i < header->num_clauses; i++) {
+		offset += sizeof (MonoObject*);
+		runtime_method->exvar_offsets [i] = offset;
+	}
```

Comment:
does this have to be in the storage for `locals` (i.e. is that some specification thing) or should we keep a separate storage for those?